rdkit/Chem/PandasTools.py

```diff
@@ -285,15 +285,15 @@ def LoadSDF(filename, idName='ID',molColName = 'ROMol',includeFingerprints=False
   for i, mol in enumerate(Chem.ForwardSDMolSupplier(f)):
```

Comment:
If you change this to: `Chem.ForwardSDMolSupplier(f,sanitize=(molColName is not None))` you'll save the sanitization step for cases where the molecule doesn't actually need to sanitized.